src/controller/org.controller/org.controller.js

```diff
@@ -69,6 +69,53 @@ async function getCna (req, res, next) {
   }
 }
 
+async function getUsers (req, res, next) {
+  try {
+    const shortName = req.ctx.org
+    const cnaShortName = req.ctx.params.shortname
+    const orgRepo = req.ctx.repositories.getOrgRepository()
+    const userRepo = req.ctx.repositories.getUserRepository()
+    const orgUUID = await orgRepo.getOrgUUID(cnaShortName)
+    let returned = false
+
+    if (!orgUUID) {
+      returned = true
+      logger.info(cnaShortName + ' CNA does not exist.')
+      return res.status(404).json(error.cnaDneParam(cnaShortName))
+    }
+
+    if (cnaShortName !== shortName && !returned) {
+      returned = true
+      logger.info({ message: cnaShortName + ' CNA can only be viewed by the owning CNA.' })
+      return res.status(403).json({ error: 'NOT_OWNER', message: 'This information can only be viewed by the owning CNA.' })
+    }
+
+    if (!returned) {
+      const result = await userRepo.aggregate([
+        {
+          $match: { org_UUID: orgUUID }
+        },
+        {
+          $project: {
+            _id: false,
+            UUID: true,
+            username: true,
+            org_UUID: true,
+            name: true,
+            active: true,
+            time: true
+          }
+        }
+      ])
+
+      logger.info(`The users of ${cnaShortName} org were sent to the user.`)
```

Comment:
We're already logging when the request was made, I don't know if we really need this statement. We can assume the request went through if we don't log any errors.

Reply:
Even if we decided to log this, it isn't JSON

Reply:
A separate issue for defining the log structure should be created. The issue should also determine the cases where logs should be included to avoid unnecessary logging.